raiden/utils/mediation_fees.py

```diff
@@ -13,8 +15,8 @@ def ppm_fee_per_channel(per_hop_fee: ProportionalFeeAmount) -> ProportionalFeeAm
     #converting-per-hop-proportional-fees-in-per-channel-proportional-fees
     for how to get to this formula.
     """
-    per_hop_ratio = per_hop_fee / 1e6
-    return ProportionalFeeAmount(round(per_hop_ratio / (per_hop_ratio + 2) * 1e6))
+    per_hop_ratio = Fraction(per_hop_fee, 10 ** 6)
+    return ProportionalFeeAmount(round(per_hop_ratio / (per_hop_ratio + 2) * 10 ** 6))
```

Comment:
Why not use a Fraction for this division too?

Reply:
It is a Fraction
```
>>> print(repr(Fraction(1) / 10))
Fraction(1, 10)
```
I'm torn between making the division more readable (current case) and making the Fraction more explicit (using `Fraction(per_hop_ratio, ...)`).

Reply:
Indeed. It's fine then.